src/components/RefinementList/RefinementList.js

```diff
@@ -6,6 +6,8 @@ import Template from '../Template.js';
 import RefinementListItem from './RefinementListItem.js';
 import isEqual from 'lodash/isEqual';
 
+import SearchBox from '../SearchBox';
```

Comment:
When I read this I thought that we were reusing our current SearchBox, turns out we do not but I understand why it's not the case. The current searchbox is only a vanilla JS widget that we made because we had weird use cases that we should not have tried to solve in the first place the way we did. `</LegacyRant>`